### PR TITLE
feat: add Material3 themes

### PR DESCRIPTION
**Pull request by m1ga**
Adding the new Material 3 themes (https://github.com/tidev/titanium_mobile/pull/13597) to the docs
**Merge after Ti 12.0.0 is released**

### CODE REVIEW

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/User_Interface_Deep_Dives/Android_UI_Components_and_Conventions/Android_Themes.md [hunk 1]
```diff
@@ -54,6 +54,21 @@ As of Titanium 10.0.0, all of the below are material based themes which support
...
 | Theme.AppDerived.Translucent.Fullscreen | 10.0.0 | Derives from `<application/>` assigned theme, applies a transparent background, excludes action bar, and excludes status bar.<br /><br />Can only be applied to activities and not the application. |
 
+## Titanium material 3 themes
+
+As of Titanium 12.0.0 you can use Material 3 dynamic color themes. Titanium provides some default themes:
+
+| Theme Name | SDK Version | Description |
+| --- | --- | --- |
+| Theme.Titanium.Material3.DayNight | 12.0.0 | Material 3 default day/night theme with dynamic colors. |
+| Theme.Titanium.Material3.DayNight.NoTitleBar | 12.0.0 | Material 3 default day/night theme with dynamic colors and no actionbar. |
```
Comment (hansemannn) [inline docs/guide/Titanium_SDK/Titanium_SDK_How-tos/User_Interface_Deep_Dives/Android_UI_Components_and_Conventions/Android_Themes.md]: `actionbar` -> `action bar` (check other places)

**Review by hansemannn — CHANGES_REQUESTED** [review]
Minor typo

**Comment by hansemannn** [comment]
I think we should merge this prior to the GA, since it already applies for the RC and nightly builds.

**Comment by m1ga** [comment]
yes, can be merged already since RC is out

**Pull request merged**